presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -73,6 +73,7 @@
     public static final String ENABLE_INTERMEDIATE_AGGREGATIONS = "enable_intermediate_aggregations";
     public static final String PUSH_AGGREGATION_THROUGH_JOIN = "push_aggregation_through_join";
     public static final String PUSH_PARTIAL_AGGREGATION_THROUGH_JOIN = "push_partial_aggregation_through_join";
+    public static final String DYNAMIC_PARTITION_PRUNING = "dynamic_partition_pruning";
```

Comment:
@arhimondr does this commit really has to be that huge? It would be very helpful if this one is split into smaller logical parts. Is there a chance for that?